test/components/plots/barPlotTests.ts

```diff
@@ -76,32 +76,6 @@ describe("Plots", () => {
         svg.remove();
       });
 
-      it("bar alignment can be changed; barPlot updates appropriately", () => {
-        barPlot.barAlignment("center");
-        var renderArea = (<any> barPlot)._renderArea;
-        var bars = renderArea.selectAll("rect");
-        var bar0 = d3.select(bars[0][0]);
-        var bar1 = d3.select(bars[0][1]);
-        assert.equal(numAttr(bar0, "width"), 150, "bar0 width is correct");
-        assert.equal(numAttr(bar1, "width"), 150, "bar1 width is correct");
-        assert.equal(numAttr(bar0, "x"), 75, "bar0 x is correct");
-        assert.equal(numAttr(bar1, "x"), 375, "bar1 x is correct");
-
-        barPlot.barAlignment("right");
-        renderArea = (<any> barPlot)._renderArea;
-        bars = renderArea.selectAll("rect");
-        bar0 = d3.select(bars[0][0]);
-        bar1 = d3.select(bars[0][1]);
-        assert.equal(numAttr(bar0, "width"), 150, "bar0 width is correct");
-        assert.equal(numAttr(bar1, "width"), 150, "bar1 width is correct");
-        assert.equal(numAttr(bar0, "x"), 0, "bar0 x is correct");
-        assert.equal(numAttr(bar1, "x"), 300, "bar1 x is correct");
-
-        assert.throws(() => barPlot.barAlignment("blargh"), Error);
-        assert.equal((<any> barPlot)._barAlignmentFactor, 1, "the bad barAlignment didnt break internal state");
-        svg.remove();
-      });
-
```

Comment:
Since barAlignment doesn't exist on bands mode ordinal scales, I'm removing this aspect for ordinal scale-backed plots (unless we want it)